platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -739,10 +753,7 @@ class Godot(private val context: Context) : SensorEventListener {
 	 */
```

Comment:
This is not part of your original changes, but to be consistent for the `isDarkModeSupported()` method above, we should just return:

```
return context.resources?.configuration?.uiMode?.and(Configuration.UI_MODE_NIGHT_MASK) != Configuration.UI_MODE_NIGHT_UNDEFINED
```